### DISCUSSION TITLE
How to hide top level header (h1) from toc?

### DISCUSSION THREAD

**Discussion by ssbarnea** [squidfunk/mkdocs-material#6492] [Q&A]
Quite often we do want to have different navigation menu entries for documents as in-document title (H1 ak "#" ) may be too long to be displayed.

The problem is that when using TOC functionality, we get an extra level that is not desired as it is mainly the document itself.

![](https://sbarnea.com/ss/mkdocs-material-toc-integrate-title.png)

Is there a way to avoid this issue, other than ensuring that all the documents do not have a top level H1(#) entry? -- not having this is quite confusing especially as the filename itself could happen to be named "index.md" or "README.md" if is an index page.

**Comment by kamilkrzyskow** [comment]
Hello,
if I understand you correctly you want to remove the `Ansible Development Tools (ADT)` title from the toc on the left?
It's the clickable navigation link for the page, so if you'd remove it you want the `Introduction` below it to be the clickable link? 
I'm not sure if that's possible, but will research it later 🤔It would create a sort of double `Introduction` from the section heading and `Introduction` nav link from the `H2` tag, which could be a bit confusing. 

Also to make it clear the nav screenshot on the right shows a `Introduction` section name with a page underneath it, so the page uses the `H1` tag as the tile.
To change the name of the page you can add 
```md
---
title: New title
---
```
at the top of your `introduction.md` file.
Or:
```yaml
# ...
- Introduction:
  - New title: introduction.md
# ...
```
